recipe/meta.yaml

```diff
@@ -11,8 +11,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
-  script: python -m pip install . -vv --no-deps
+  number: 1
```

Comment:
Please note that build 0 lives in the sf* channel so we cannot use build 0 in defaults.

Reply:
This isn't a new package. It's new in defaults, but it was already built for snowflake. Since we can't have overlap between these new channels we had to bump the build number

Reply:
Thanks for clarification